packages/terra-framework-docs/src/terra-dev-site/doc/worklist-data-grid/Examples.2/RowSelection.jsx

```diff
@@ -83,6 +145,16 @@ const RowSelection = () => {
     setSelectedRows([]);
   };
 
+  const enableSelectableRows = (rowIds) => {
+    const rowsToSelect = rows.filter(e => rowIds.indexOf(e.id) >= 0);
+    if (!rowSelectionModeRef.current.checked) {
+      rowSelectionModeRef.current.checked = true;
+    }
+    rowsToSelect.forEach(r => { r.isSelected = true; });
```

Comment:
Did you mean to use "==="?  If "isSelected" is a Boolean value, you don't need the extra comparison.

Reply:
No, it's assignment.